### DISCUSSION TITLE
Simulate Holding the Touch ID or Power Button to launch Startup Security Utility

### DISCUSSION THREAD

**Discussion by mrpepik** [cirruslabs/tart#984] [Q&A]
Hi Everyone,

I have a Monterey VM that I wanted to test OpenZFS on MacOS.  However to fully install it, I need to hold the Power Button or Touch ID on power on, so I can launch the Startup Security Utility to enable the kernel extension.

Is there a way with Tart to do this?  

Thanks in advance for the help.

MrPepik

**Comment by fkorotkov** [comment]
Do you want to boot in recovery mode? If so, `tart run --recovery VM_NAME` will do the trick.

**Comment by mrpepik** [comment]
Let me try that.

Thank you

**Comment by mrpepik** [comment]
Fedor,

Thank you, that allowed me to access the security utility.
